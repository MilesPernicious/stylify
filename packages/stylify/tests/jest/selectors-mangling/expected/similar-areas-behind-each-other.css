.a{
	color: blue
}
.b{
	font-size: 12px
}
.c{
	color: green
}
.d{
	font-size: 34px
}
