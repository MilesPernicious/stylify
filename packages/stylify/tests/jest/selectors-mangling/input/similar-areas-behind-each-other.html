<div onload="
addAttribute('color:blue font-size:12px', 'class:list');addAttribute('color:green font-size:34px', 'class')
">
</div>
