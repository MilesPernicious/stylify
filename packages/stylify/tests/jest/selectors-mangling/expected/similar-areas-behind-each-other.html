<div onload="
addAttribute('a b', 'class:list');addAttribute('c d', 'class')
">
</div>
